### ISSUE TITLE
SingleThreadEventLoopTest failure on Windows 7 x64

### ISSUE THREAD

**Issue by trustin** [netty/netty#3817]
```
SingleThreadEventLoopTest.testDeregisterWithScheduleAtFixedRateTask:585
  ->testDeregisterWithPeriodicScheduleTask:650
  ->verifyTimestampDeltas:662 delta: 187164722, minDelta: 190000000
```

**Comment by trustin** [comment]
Reliably reproduceable on my home PC

**Comment by Scottmitch** [comment]
May be related to https://github.com/netty/netty/issues/3231

**Comment by trustin** [comment]
Fixed by 49988a68212939423ae5e98d8b4ff60aeea0e980
